### ISSUE TITLE
BUG - 1.09 RC4 Lost Models when updating from 1.08 to 1.09RC4

### ISSUE THREAD

**Issue by Joetrin** [FrSkyRC/ETHOS-Feedback-Community#476]
After updating form 1.08 to 1.09 only 1 model shows, please see the photos.  There are 4 models in the models folder.  But when selecting aircraft, only 1 model shows.

![20210611_215450](https://user-images.githubusercontent.com/19473795/121761992-5cf68e00-cb01-11eb-9a56-454822902929.jpg)
![20210611_215512](https://user-images.githubusercontent.com/19473795/121761997-6122ab80-cb01-11eb-9b70-9a06842b6592.jpg)
![20210611_215520](https://user-images.githubusercontent.com/19473795/121761998-61bb4200-cb01-11eb-8f4d-e7e43dd2d889.jpg)
![20210611_215456](https://user-images.githubusercontent.com/19473795/121761999-62ec6f00-cb01-11eb-9026-0cb976b82881.jpg)

**Comment by Hobby4life** [comment]
I have seen 1 model lately dissapear. It was a "others" model

**Comment by Joetrin** [comment]
The strange thing about all this is, all were airplanes and 3 of the models were copies of one that was lost.  

I created a model for my 102" ARS300 and copied it for the smaller 91"

**Comment by bsongis-frsky** [comment]
I need one of those models to understand why it disappears!

**Comment by Joetrin** [comment]
Hi Bernard,

Only Model 6 is displaying on the screen. Please see all the model files attached.
Please rename the .txt to .bin, the platform is not accepting .bin.

Merci


[model07.txt](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/6647860/model07.txt)
[model08.txt](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/6647861/model08.txt)
[model05.txt](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/6647862/model05.txt)
[model06.txt](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/6647863/model06.txt)

**Comment by bsongis-frsky** [comment]
I am not sure those models have been created with the official 1.0.8, I wonder if they comes from an intermediate firmware just before (I would say the day just before) the release has been posted

**Comment by bsongis-frsky** [comment]
Or perhaps from a simulator?

**Comment by bsongis-frsky** [comment]
Those models have been created from a test firmware (which I compiled myself), now I understand!

**Comment by Joetrin** [comment]
Hi Bernard,

I have never used the simulator, these models were created in 1.07 and worked in 1.08 and 1.08 Pre-Release.  They all worked fine.

When I updated to 1.09 RC4, that is when the issue occured, and only Model06 appeared.
The strange symptom occured after the update, the new model wizard came on the radio immediately after reboot, then never showed again.

I did not install any other RC's from 1.09, Just the RC4

Hope this helps.
